Akinator.Api.Net/Utils/StringExtensions.cs

```diff
@@ -1,19 +1,21 @@
 ﻿using System;
+using System.IO;
 using System.Text;
-using System.Web;
 
 namespace Akinator.Api.Net.Utils
 {
     public static class StringExtensions
     {
+        static char[] hexChars = "0123456789abcdef".ToCharArray();
+
         public static string UrlEncode(this string input)
         {
             if (input == null)
             {
                 throw new ArgumentNullException(nameof(input));
             }
 
-            var temp = HttpUtility.UrlEncode(input, Encoding.UTF8).ToCharArray();
+            var temp = UrlEncode(input, Encoding.UTF8).ToCharArray();
```

Comment:
@janniksam why did you have removed `HttpUtility.UrlEncode` and used another not official code?

Reply:
HttpUtitlity wasn't there back in 1.4. And I did not find any lightweight nuget package to use.

Reply:
@janniksam 
![IMG_20200411_110501](https://user-images.githubusercontent.com/34550324/79039844-5eb68000-7be4-11ea-86e7-77be3ce52bb8.jpg)

Reply:
So you can use WebUtility instead of HttpUtility

Reply:
thank you, good point. Done.